dev/com.ibm.ws.kernel.service/src/com/ibm/ws/kernel/service/util/CpuInfo.java

```diff
@@ -85,14 +91,19 @@ private CpuInfo() {
             phase.addMultiThreadedHook(activeTask);
         }
         int runtimeAvailableProcessors = Runtime.getRuntime().availableProcessors();
-        int fileSystemAvailableProcessors = getAvailableProcessorsFromFilesystem();
+        float fileSystemAvailableProcessors = getAvailableProcessorsFromFilesystemFloat();
 
         if (fileSystemAvailableProcessors <= 0 || fileSystemAvailableProcessors > runtimeAvailableProcessors) {
-            AVAILABLE_PROCESSORS.set(runtimeAvailableProcessors);
+            AVAILABLE_PROCESSORS_INTEGER.set(runtimeAvailableProcessors);
+            AVAILABLE_PROCESSORS_FLOAT.set(runtimeAvailableProcessors * 100);
         } else {
-            AVAILABLE_PROCESSORS.set(fileSystemAvailableProcessors);
+            AVAILABLE_PROCESSORS_INTEGER.set(roundUpToNextInt(fileSystemAvailableProcessors));
+            AVAILABLE_PROCESSORS_FLOAT.set((int) fileSystemAvailableProcessors * 100);
         }
 
+        System.out.println("***gjd*** AVAILABLE_PROCESSORS_INTEGER: " + AVAILABLE_PROCESSORS_INTEGER.get());
+        System.out.println("***gjd*** AVAILABLE_PROCESSORS_LONG:    " + AVAILABLE_PROCESSORS_FLOAT.get() / 100);
```

Comment:
```suggestion
        System.out.println("***gjd*** AVAILABLE_PROCESSORS_LONG:    " + AVAILABLE_PROCESSORS_FLOAT.get() / 100.0);
```

Reply:
The string should be changed to FLOAT instead of LONG, but this will be removed so it could stay for now.